Python/compile.c

```diff
@@ -7280,10 +7284,10 @@ makecode(struct compiler *c, struct assembler *a, PyObject *constslist,
     if (localsplusnames == NULL) {
         goto error;
     }
-    if (_PyCode_InitLocalsPlusKinds(nlocalsplus, &localspluskinds) < 0) {
+    localspluskinds = compute_localsplus_info(c, localsplusnames);
+    if (localspluskinds == NULL) {
```

Comment:
This certainly becomes simpler with the signature I suggested above for `compute_localsplus_info()`.

```c
        if (compute_localsplus_info(c, nlocalsplus, &localsplusnames, &localspluskinds) != 0) {
            goto error;
        }
```

Reply:
Should be ready for another review.